### DISCUSSION TITLE
[Feature] Enable multiselection in the show/hide faces page

### DISCUSSION THREAD

**Discussion by gga-kialo** [immich-app/immich#5142] [Feature Request]
### The feature

Every time I upload new pictures, my `/people` page gets cluttered with faces of background people, and even statues, drawn faces, dolls... Even though named faces appear on top, I prefer hiding all others and keeping only those that I may want to ever search for. I currently have to do so by clicking one by one over them on the hide/show page. It would be better if I could easily hide them all at once by multi-selecting with the shift key, or dragging over them (https://github.com/immich-app/immich/discussions/4935). Having an option to hide newly recognized faces by default would be even nicer.

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by upsetdog** [comment]
Perhaps increasing the min faces option in admin>machine learning>facial recognition would help you for now (mine is set to 3)?

**Reply by gga-kialo** [reply]
Great tip! I didn't know about that setting. Thanks.
I still think that multiselection would be a nice addition, but now it is no longer a feature I desperately need. :)

**Reply by chodthewacko** [reply]
> Perhaps increasing the min faces option in admin>machine learning>facial recognition would help you for now (mine is set to 3)?

I have lots of pictures of school events and all the faces of random students/classmates have a large number of matching images.   min faces doesn't help here.

There is a pending request for 'select/hide all unnamed faces' which would help a lot.
I made this comment to also suggest 'invert selection', which would save a LOT of clicks.  It's then two clicks (select my kid, then invert) versus X-1 number of clicks to manually select everyone that isn't my kid.

**Comment by Biepa** [comment]
Would love to see that as well.

**Comment by Biepa** [comment]
I just saw there is this button where it's much easier to hide multiple people.
![image](https://github.com/user-attachments/assets/2b719d17-8911-490a-bbed-a4752270cd70)
